Source: Bernd Steinhauser <berniyh@exherbo.org>
Reason: Fails with cmake >= 2.8.6
Upstream: Reported, Bug 288712. 

From 31f08731cb21d206b17653f4519f6c46a14f2648 Mon Sep 17 00:00:00 2001
From: Bernd Steinhauser <berniyh@exherbo.org>
Date: Sun, 11 Dec 2011 13:01:55 +0100
Subject: [PATCH] Don't try to find BoostConfig.cmake.

---
 CMakeLists.txt |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 416c0c6..4bc2ed2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,6 +18,7 @@ if(${CMAKE_BUILD_TYPE} MATCHES coverage)
   set (CMAKE_CXX_FLAGS "${KDE4_ENABLE_EXCEPTIONS}" )
 endif()
 
+set (Boost_NO_BOOST_CMAKE TRUE)
 find_package(Boost REQUIRED)
 macro_log_feature(Boost_FOUND "Boost C++ Libraries" "Required to build Rocs" "http://www.boost.org" TRUE "" "")
 
-- 
1.7.7.4

